src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -977,10 +1004,11 @@ pub struct GraphEditorModel {
     pub logger         : Logger,
     pub display_object : display::object::Instance,
     pub scene          : Scene,
-    pub project_name   : component::ProjectName,
+    pub breadcrumbs    : component::Breadcrumbs,
     pub cursor         : cursor::Cursor,
     pub nodes          : Nodes,
     pub edges          : Edges,
+    network            : frp::Network,
```

Comment:
This causes huge memory leaks! You cannot use `network` inside of model! Network keeps references to FRP nodes, which keep references to model! This line was not here during my previous PR. Danilo, we have to be careful here. We cannot introduce such big memory leaks during PR fixing. You have to be very careful about memory leaks.